lib/Elastica/Type.php

```diff
@@ -358,17 +358,18 @@ public function search($query = '', $options = null)
     /**
      * Count docs by query.
      *
-     * @param string|array|\Elastica\Query $query Array with all query data inside or a Elastica\Query object
-     *
-     * @return int number of documents matching the query
+     * @param string|array|\Elastica\Query $query       Array with all query data inside or a Elastica\Query object
+     * @param ResultSet                    $fullResult (default = false) By default only the total hit count is returned. If set to true, the full ResultSet including aggregations is returned.
+     * 
+     * @return int number of documents matching the query|ResultSet
      *
      * @see \Elastica\SearchableInterface::count
      */
-    public function count($query = '')
+    public function count($query = '', $fullResult = false)
```

Comment:
@vinusebastian Idea was not necessarly to close this one, but have a chat with @intech on the best approach :-)